src/librustdoc/doctest.rs

```diff
@@ -897,6 +933,61 @@ fn partition_source(s: &str, edition: Edition) -> (String, String, String) {
     (before, after, crates)
 }
 
+pub(crate) struct RustdocTestOptions {
```

Comment:
```suggestion
pub(crate) struct IndividualTestOptions {
```
to mirror [`GlobalTestOptions`](https://github.com/rust-lang/rust/blob/7776678bef8ca178b63d4aa50f734d72f419757d/src/librustdoc/doctest.rs#L44)

Reply:
Good idea!